<?xml version="1.0"?>
<launch>
 
  
  
  <include file="$(find doosan_moveit_config)/launch/demo_gazebo.launch">
    <!-- robot_description is loaded by gazebo.launch, to enable Gazebo features -->
    <arg name="world_name" value="$(find doosan_gazebo_sim)/worlds/animated_box.world" />
  </include>

  <!-- Launch the tracking node -->
  <node pkg="visp_auto_tracker" type="visp_auto_tracker" name="visp_auto_tracker" output="screen">
    <param name="model_path" value="$(find visp_auto_tracker)/models" />
    <param name="model_name" value="pattern" />
    <param name="debug_display" value="True" />
    <!-- param name="code_message" value="Ver1" / -->
    
    <remap from="/visp_auto_tracker/camera_info" to="robotic_arm/camera1/camera_info"/>
    <remap from="/visp_auto_tracker/image_raw" to="robotic_arm/camera1/image_raw"/>
  </node>



</launch>
